webdriver/bidi/permission.json

```
@@ -34,7 +34,7 @@
         "setPermission": {
           "__compat": {
             "description": "`permission.setPermission` command",
-            "spec_url": "https://www.w3.org/TR/permissions/#webdriver-bidi-command-permissions-setPermission",
+            "spec_url": "https://www.w3.org/TR/permissions/#dfn-set-permission-0",
```

This is actually just the same version issue, and we can link the same fragment in the draft:

```suggestion
            "spec_url": "https://w3c.github.io/permissions/#webdriver-bidi-command-permissions-setPermission",
```